api/status/urls.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+Routes for api status endpoints
+"""
+from django.conf.urls import patterns, include, url
+from rest_framework import routers
+
+from api.status import views
+
+router = routers.DefaultRouter(trailing_slash=False)
+router.register(r'celery', views.CeleryViewSet, base_name='celery')
+
+api_status_urls = router.urls
+urlpatterns = patterns('', url(r'^', include(api_status_urls)),)
```

Comment:
By the way - `patterns(...)` is being deprecated in Django 1.9. I believe the suggestion is for this to be a simple Python list or `url( , )`:
```python
urlpatterns = [
   url(r'^', include(api_status_urls))
] 
```
source: [troposphere/urls.py](https://github.com/iPlantCollaborativeOpenSource/troposphere/pull/302/files)